components/Map.tsx

```diff
@@ -34,27 +36,28 @@ const Map: React.FunctionComponent<MapProps & ViewportStateProps> = ({
       onViewportChange={(newViewport) => setViewport(newViewport)}
       {...viewport}
     >
-      {orgs
-        ? orgs.map((org) => {
-            return org.lat && org.long ? (
-              <div key={org.id}>
-                <Marker latitude={org.lat} longitude={org.long}>
+      {objs
+        ? objs.map((obj: Organization | OrganizationEvent) => {
+            return obj.lat && obj.long ? (
+              <div key={obj.id}>
+                <Marker latitude={obj.lat} longitude={obj.long}>
                   <span
-                    onClick={() => setSelectedOrg(org)}
+                    onClick={() => setSelectedObj(obj)}
                     role="img"
                     aria-label="push-pin"
                   >
                     📌
                   </span>
                 </Marker>
-                {selectedOrg?.id === org.id ? (
+                {selectedObj?.id === obj.id ? (
                   <Popup
-                    onClose={() => setSelectedOrg(null)}
+                    onClose={() => setSelectedObj(null)}
                     closeOnClick
-                    latitude={org.lat}
-                    longitude={org.long}
+                    latitude={obj.lat}
+                    longitude={obj.long}
                   >
-                    {org.name}
+                    {typeof obj}
```

Comment:
you can use typeguard function like below
```ts
const isOrganization = (
    obj: Organization | OrganizationEvent
  ): obj is Organization => {
    if ((obj as Organization).name) return true;
    return false;
  };
```

for reference: https://medium.com/ovrsea/checking-the-type-of-an-object-in-typescript-the-type-guards-24d98d9119b0